frontend/console/src/api/modules/use-stream-modules.ts

```diff
@@ -0,0 +1,46 @@
+import { Code, ConnectError } from '@connectrpc/connect'
+import { useQuery, useQueryClient } from '@tanstack/react-query'
+import { useClient } from '../../hooks/use-client'
+import { useVisibility } from '../../hooks/use-visibility'
+import { ConsoleService } from '../../protos/xyz/block/ftl/v1/console/console_connect'
+import type { Module } from '../../protos/xyz/block/ftl/v1/console/console_pb'
+
+const streamModulesKey = 'streamModules'
+const updateIntervalMs = 60 * 1000
```

Comment:
Should we pass this in and default it? Also, is this the correct number of nano seconds (I might have had this wrong before too 😂)?

Math, mostly for me...
1 nano = 1,000,000 milliseconds
```
60 * 1000 * 1000 = 60,000,000
```
So I think this is updating at `60ms` which might be faster than we want.

Reply:
Oof thanks for catching that nano issue!

In retrospect, I'm going to just remove this and leave it totally controlled by the server side. It seems like something we'll be able to optimize a lot more cleanly without having to _also_ conform to potentially several distinct client-side refresh thresholds...